kyo-prelude/shared/src/main/scala/kyo/kernel/Loop.scala

```diff
@@ -141,9 +148,12 @@ object Loop:
         loop(input1, input2, input3, input4)()
     end apply
 
-    inline def indexed[O, S](inline run: Int => Outcome[Unit, O] < S)(using inline _frame: Frame): O < S =
+    inline def indexed[O, S](inline run: Int => Outcome[Unit, O] < S)(using
```

Comment:
Not sure why the compiler has inference issues if I add the `Safepoint` to `run` specifically in this method.

Reply:
Is it still an issue if the method is not inline?

Reply:
yep, same thing:

```
[error] -- [E081] Type Error: /Users/fwbrasil/workspace/kyo/kyo-prelude/shared/src/main/scala/kyo/Kyo.scala:104:39
[error] 104 |                        Loop.indexed { idx =>
[error]     |                                       ^^^
[error]     |                       Missing parameter type
[error]     |
[error]     |                       I could not infer the type of the parameter idx
[error]     |                       Expected type for the whole anonymous function:
[error]     |                         Any
```